src/praisonai/praisonai/cli/main.py

```
@@ -4379,23 +4383,43 @@ def level_based_approve(function_name, arguments, risk_level):
+                        
+                        if direct:
+                            # Pass-through proxy (original behavior, preserved as escape hatch)
+                            ext_console.print(f"[bold cyan]🔌 Using external agent (direct): {external_agent_name}[/bold cyan]")
                             import asyncio
                             try:
                                 result = asyncio.run(integration.execute(prompt))
                                 ext_console.print(f"\n[bold green]Result from {external_agent_name}:[/bold green]")
                                 ext_console.print(result)
-                                # Return empty string to avoid duplicate printing by caller
                                 return ""
                             except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)